<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">

    <node name="rtk_type_printer" pkg="mrs_odom" type="rtk_type_printer" output="screen" >
      
      <!-- Subscribers --> 
      <remap from="~diagnostics" to="rtk_gps/diagnostics" />

    </node>

  </group>

</launch>
